src/renderer/components/dialogs/ViewGroup.tsx

```diff
@@ -247,22 +243,14 @@ function ViewGroupInner(
             onClose={onClose}
           />
           <DialogBody>
-            <DialogContent>
-              <div className='group-settings-container'>
-                <ClickForFullscreenAvatarWrapper filename={groupImage}>
-                  <Avatar
-                    displayName={groupName}
-                    avatarPath={groupImage}
-                    color={chat.color}
-                    wasSeenRecently={chat.wasSeenRecently}
-                    large
-                  />
-                </ClickForFullscreenAvatarWrapper>
-                <p className='group-name' style={{ marginLeft: '17px' }}>
-                  <p className='trucated-name'>{groupName}</p>
-                  {chat.isProtected && <InlineVerifiedIcon />}
-                </p>
-              </div>
+            <DialogContent paddingBottom>
+              <ProfileInfoHeader
+                avatarPath={groupImage ? groupImage : undefined}
+                color={chat.color}
+                displayName={groupName}
+                isVerified={chat.isProtected}
+                wasSeenRecently={chat.wasSeenRecently}
```

Comment:
groups don't have wasSeenRecently (green dot)

Reply:
I'll make it optional to reflect that in the API